lib/adagrams.rb

```diff
@@ -0,0 +1,151 @@
+letter_quantities = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4,
+  E: 12,
+  F: 2,
+  G: 3,
+  H: 2,
+  I: 9,
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1,
+}
+$letter_pool = []
+letter_quantities.each do |letter, quantity|
+  quantity.times do
+    $letter_pool << letter.to_s
+  end
+end
```

Comment:
You all populated a variable representing the letter pool as a global variable! In some ways, this makes sense, but in general, we'll avoid using global variables. Our hope was top put the above few lines within the `draw_letters` method, since that's the only time you'll use this variable.